web-client/src/views/AdvancedSearch/PractitionerSearchResults.tsx

```diff
@@ -27,9 +27,9 @@ export const PractitionerSearchResults = connect(
               {practitionerSearchHelper.showPaginator && (
                 <Paginator
                   breakClassName="hide"
-                  forcePage={practitionerSearchHelper.activePage || 0}
+                  forcePage={practitionerSearchHelper.activePage}
```

Comment:
I think these still need to be defaulted in the helper.

Reply:
it's being defaulted in the helper in the block starting on line 45, except `activePage`, which is defaulted in `defaultAdvancedSearchFormAction` because it's shared across a few components. would it make sense to default it another way?

Reply:
Hmmm I only see it referenced in the helper and the tsx. To me, I don't see a reason it needs to be a value on state.